src/applications/gi/components/profile/EstimateYourBenefitsForm.jsx

```diff
@@ -1019,7 +1022,14 @@ class EstimateYourBenefitsForm extends React.Component {
         </ul>
         <button
           className="calculate-button"
-          onClick={this.props.updateEstimatedBenefits}
+          onClick={
```

Comment:
can this be pulled out into a variable for easier readability?

Reply:
Sure